boefjes/boefjes/plugins/kat_lapje/main.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from typing import List, Tuple, Union
+
+import docker
+
+from boefjes.job_models import BoefjeMeta
+from boefjes.plugins.helpers import get_file_from_container
+
+logger = logging.getLogger(__name__)
+
+OCI_IMAGE = "lapje"
```

Comment:
Looking at this tag versus e.g. the masscan boefje and its custom github workflow: I suppose this should just work by building the image with the `lapje` tag locally?

UPDATE:
Yep this works:
```shell
docker build -t lapje boefjes/images/lapje
```

![image](https://github.com/minvws/nl-kat-coordination/assets/46660228/e6205dd0-8b0b-4e9e-8d88-a3a43ad66a02)